ScoutSuite/providers/gcp/facade/facade.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is implicit with the recent python versions (you'll notice we dropped that line in new files in the other PRs).